client/app/lib/base_view.es6

```diff
@@ -1,12 +1,12 @@
-module.exports = class BaseView extends Backbone.View {
-  constructor() {
-    super();
-  }
 
-  template() {}
+export default class BaseView extends Backbone.View {
+
+  template() {
+    return '';
+  }
 
   getRenderData() {
-    return { model: this.model.toJSON() };
+    return { model: JSON.stringify(this.model) };
```

Comment:
@Peltoche you are sure about this change?
Backbone models saves its attributes in an hidden field and use getter/setter to access them. If you do a JSON.stringify, I think you'll have all the things nested in an `attributes` object. Correct me if I am wrong.